dns-resolver/rootfs/etc/unbound/unbound.tmpl

```diff
@@ -0,0 +1,114 @@
+  use-caps-for-id: yes
+
+  # the time to live (TTL) value lower bound, in seconds. Default 0.
+  # If more than an hour could easily give trouble due to stale data.
+  cache-min-ttl: 3600
+
+  # the time to live (TTL) value cap for RRsets and messages in the
+  # cache. Items are not cached for longer. In seconds.
+  cache-max-ttl: 86400
+
+  # perform prefetching of close to expired message cache entries.  If a client
+  # requests the dns lookup and the TTL of the cached hostname is going to
+  # expire in less than 10% of its TTL, unbound will (1st) return the ip of the
+  # host to the client and (2nd) pre-fetch the dns request from the remote dns
+  # server. This method has been shown to increase the amount of cached hits by
+  # local clients by 10% on average.
+  prefetch: yes
+
+  ## Unbound Optimization and Speed Tweaks ###
+
+  # use all CPUs
+  num-threads: 4
+
+  # the number of slabs to use for cache and must be a power of 2 times the
+  # number of num-threads set above. more slabs reduce lock contention, but
+  # fragment memory usage.
+  msg-cache-slabs: 8
+  rrset-cache-slabs: 8
+  infra-cache-slabs: 8
+  key-cache-slabs: 8
+
+  # Increase the memory size of the cache. Use roughly twice as much rrset cache
+  # memory as you use msg cache memory. Due to malloc overhead, the total memory
+  # usage is likely to rise to double (or 2.5x) the total cache memory.
+  rrset-cache-size: 256m
+  msg-cache-size: 128m
+
+  # buffer size for UDP port 53 incoming (SO_RCVBUF socket option). This sets
+  # the kernel buffer larger so that no messages are lost in spikes in the traffic.
+  # sysctl -w net.core.rmem_max=1048576
+  # so-rcvbuf: 1m
+
+  # If nonzero, unwanted replies are not only reported in statistics, but also
+  # a running total is kept per thread. If it reaches the threshold, a warning
+  # is printed and a defensive action is taken, the cache is cleared to flush
+  # potential poison out of it.  A suggested value is 10000000, the default is
+  # 0 (turned off). We think 10K is a good value.
+  unwanted-reply-threshold: 10000
+
+  # IMPORTANT FOR TESTING: If you are testing and setup NSD or BIND  on
+  # localhost you will want to allow the resolver to send queries to localhost.
+  # Make sure to set do-not-query-localhost: yes . If yes, the above default
+  # do-not-query-address entries are present.  if no, localhost can be queried
+  # (for testing and debugging). 
+  do-not-query-localhost: no
+
+  # Should additional section of secure message also be kept clean of unsecure
+  # data. Useful to shield the users of this validator from potential bogus
+  # data in the additional section. All unsigned data in the additional section
+  # is removed from secure messages.
+  val-clean-additional: yes
+
+  # Faster UDP with multithreading
+  so-reuseport: yes
+
+  statistics-interval: 0
+  extended-statistics: yes
+  # set to yes if graphing tool needs it
+  statistics-cumulative: no 
+        
+forward-zone:
```

Comment:
If the cache contains the host from the query (not expired) it will return the content from the cache. Any other case the query is forwarded to the "." zone.